app/views/changesets/show.html.erb

```diff
@@ -33,7 +33,7 @@
         <li id="c<%= comment.id %>">
           <small class='text-muted'>
             <%= t comment.visible ? ".comment_by_html" : ".hidden_comment_by_html",
-                  :time_ago => friendly_date_ago(comment.created_at),
+                  :time_ago => link_to(friendly_date_ago(comment.created_at), "https://www.openstreetmap.org/changeset/#{@changeset.id}##{comment.id}"),
```

Comment:
```suggestion
                  :time_ago => link_to(friendly_date_ago(comment.created_at), "/changeset/#{@changeset.id}##{comment.id}"),
```
But there are probably more rail-ish ways to create that url....

Reply:
You can generally get the path to an object using `*_path` helpers, e.g. `changeset_path`. So changeset_path(changeset) will link to the changeset. If you need an anchor (i.e. a `#something` in the url) then have a look through the views for other places that use the word "anchor", like for diary entry comments.

You will also find that in some circumstances you can simplify the `link_to("foo", changeset_path(changeset)) further, since (most of the time) rails knows how to build a path for an object, e.g. `link_to("foo", changeset)`. You can check other places in the existing codebase to learn what works.

Reply:
Url fragments are incorrect. They are `#c#{comment.id}`, not `##{comment.id}`.

```suggestion
                  :time_ago => link_to(friendly_date_ago(comment.created_at), :anchor => "c#{comment.id}"),
```